features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/modules/policy/apiPolicy.jag

```diff
@@ -527,4 +527,24 @@ var objectToJson = function (policy) {
     }
     return apiPolicy;
 };
+
+var validateRoles = function(roles,username) {
+    var validRole,
+    provider = jagg.module("manager").getAPIProviderObj();
+    try {
+        validRole = provider.validateRoles(roles,username);
+        if (log.isDebugEnabled()) {
```

Comment:
No need to check log.isDebugEnabled() when there is no string concatenation in the debug message